internal/testrunner/runners/pipeline/test_config.go

```diff
@@ -12,11 +12,16 @@ import (
 	"path/filepath"
 
 	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/testrunner"
 )
 
 const configTestSuffix = "-config.json"
 
 type testConfig struct {
+	// Skip allows this test to be skipped.
+	Skip *testrunner.SkipConfig `config:"skip"`
```

Comment:
I see that you placed `Skip` section in every config file, so I was wondering if it's a good idea to create a `commonTestConfig`?

Reply:
In 4e25215, I introduced a `SkippableConfig` struct that could be embedded in any test config structs that allow skipping. Let me know what you think of this approach.

Reply:
Composition, it should do the trick as well :)